### PR TITLE
JUnit Tests

### PR DESCRIPTION
**Pull request by kristiyanpetkov**

### CODE REVIEW

**Comment by mgenov** [comment]
workspace.xml and vcs.xml should not be committed.

**Comment by mgenov** [comment]
Title of this PR doesn't describe the change that was made.

**Comment by mgenov** [comment]
PR title is not used for notification. 

You can mention persons using @ character @kristiyanpetkov 

When you are ready with your fix, you can add comment to the issue, by mentioning the person need to be notified for your comment if he is not already subscribed to this PR/issue.

**Comment by mgenov** [comment]
I'm not sleeping on this PR to see that title was changed and be able to see your code.

File: src/Warehouse.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            dispay.setDisplayMessage(quantityToDeliver + " bottles of " + productToDeliver + " were delivered in the warehouse");
+        } else {
+            dispay.setDisplayMessage("The quantity to deliver is more than the Warehouse can accept");
+        }
+    }
+
+    public void sellProducts(String productForSale, int quantityForSale) {
+        Product product = products.get(productForSale);
+        if (product == null) {
+            dispay.setDisplayMessage("Product not avaible!");
```
Comment (mgenov) [inline src/Warehouse.java]: display.displayMessage is more then enough I think.

File: test/ArrayTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import org.junit.Test;
```
Comment (mgenov) [inline test/ArrayTest.java]: no package is used, please put this test and opposite class in a proper package.

File: src/Warehouse.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import java.util.Map;
+
+/**
+ * Created by clouway on 15-9-11.
+ */
+public class Warehouse {
+    private final Display dispay;
+    Map<String, Product> products;
```
Comment (mgenov) [inline src/Warehouse.java]: why map has not visibility modifier ?

File: test/ArrayTest.java (hunk 2 above)
Comment (kristiyanpetkov) [inline test/ArrayTest.java]: Done.

File: src/Warehouse.java (hunk 3 above)
Comment (kristiyanpetkov) [inline src/Warehouse.java]: i just missed it. already fixed!

File: src/Warehouse.java (hunk 1 above)
Comment (kristiyanpetkov) [inline src/Warehouse.java]: Fixed!

File: test/array/ArrayTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
+package array;
```
Comment (mgenov) [inline test/array/ArrayTest.java]: Maybe something like "com.clouway.collections" is more appropriate.

Comment (mgenov) [inline test/array/ArrayTest.java]: You can see the example from first task: https://github.com/clouway/teamspace/wiki/Introducing%20Java

Comment (kristiyanpetkov) [inline test/array/ArrayTest.java]: I hope it is better now.

**Comment by mgenov** [comment]
LGTM !

**Comment by mgenov** [comment]
Why this task is still not in upstream ?

**Comment by kristiyanpetkov** [comment]
Pushing it in the upstream is still a big mess for me. I'm working on it.

**Pull request merged**